docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,1035 @@
+
+Also the NodePool Controller calling tha AWS Cluster API provider, will create some objects in AWS which are these ones:
+
+- **Diagram:** The diagram shows the NodePool resultant objects created into the AWS Infrastructure.
+
+```mermaid
+flowchart LR
+    style HO fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style EC2 fill:#FEA175,stroke:#333,stroke-width:3px
+    style IAM fill:#FEA175,stroke:#333,stroke-width:3px
+    subgraph OCP["Openshift Management Cluster"]
+        direction LR
+        HO[fa:fa-server Hypershift Operator]
+    end
+    HO ==> EC2
+    HO ==> IAM
+    EC2I1 -.-> IAMR
+    EC2I2 -.-> IAMR
+    EC2I1 -.-> IAMP
+    EC2I2 -.-> IAMP
+    subgraph ACP["AWS Cloud Provider"]
+        direction LR
+        subgraph EC2
+            EC2I1[EC2 Instance]
+            EC2I2[EC2 Instance]
+        end
+        subgraph IAM
+            IAMR[Role Policy]
+            IAMP[Profile]
+        end
+    end
+
+```
+
+!!! note "Legend"
+    - **Orange:** OCP objects directly related with AWS.
+    - **Cyan:** Provider agnostic OCP objects.
+    - The *dotted* lines are associations among AWS Components created by the CLI or the Operator
+    - The **thick** lines are objects created by the CLI into AWS platform and needed by the Hypershift Operator to work properly
+
+## AWS Components and Hypershift
+
+As you've seen, we've been showing all the components created, managed, not managed, etc... in the Hypershift life cycle. In this section we want to dissect who creates what and what not in this 4 stages:
+
+- Pre-required and Unmanaged Hosted Cluster AWS account
+- Infra managed by hypershift in Management AWS account
+- Infra managed by hypershift in Hosted Cluster AWS account
+- Infra managed by kubernetes in Hosted Cluster AWS account
+
+#### Pre-required and Unmanaged Infra in Hosted Cluster AWS account
+
+=== "Public"
+    - 1 VPC
+    - 1 DHCP Options
+    - 1 Public Subnet (for Public HostedCluster)
+    - 1 Internet Gateway
+    - 1 NAT Gateway
+    - 1 Security Group (Worker Nodes)
+    - 1 Route Tables (Public)
+    - 1 Ingress Service Load Balancer
+
+=== "Private"
+    - 1 VPC
+    - 1 DHCP Options
+    - 1 Private Subnet (for Private HostedCluster)
+    - 1 Internet Gateway
+    - 1 NAT Gateway
+    - 1 Security Group (Worker Nodes)
+    - 1 Route Tables (Private)
+    - 1 Private Link
+
+=== "PublicAndPrivate"
+    - 1 VPC
+    - 1 DHCP Options
+    - 2 Subnets
+        - 1 Private Subnet (for Private HostedCluster)
+        - 1 Public Subnet (for Public HostedCluster)
+    - 1 Internet Gateway
+    - 1 NAT Gateway
+    - 1 Security Group (Worker Nodes)
+    - 2 Route Tables (1 Private, 1 Public)
+    - 2 Hosted Zones
+        - 1 Ingress Service Load Balancer (for Public Hosted Clusters)
+        - 1 Private Link (for Private Hosted Clusters)
+
+#### AWS Infra Managed by Hypershift
+
+=== "Management Cluster"
+    - Kube API Server Load Balancer
```

Comment:
As below can we present this as "aws resource type - use case", e.g "ELB - used for Kube API Server Load Balancer"